tests/docker/tests_single_server/run-tests.sh

```diff
@@ -26,9 +26,8 @@ dart test test/websocket_test.dart
 dart test test/types_test.dart
 dart test test/migrations_test.dart
 dart test test/exception_test.dart
-dart test test/entity_relations/table_test.dart
-dart test test/entity_relations/query_test.dart
-dart test test/entity_relations/sql_query_test.dart
+dart test test/entity_relations --concurrency=1 # Tests in folder can not be run in parallel
+dart test test/column_operations --concurrency=1 # Tests in folder can not be run in parallel
```

Comment:
Suggestion (other PR?): I think it would make sense to extract these e2e tests in a different package and streamline the script running them so we don't have to manually add the tests all the time. 

We should extract them so that we have different places for e2e tests and unit tests which makes it much easier to manage during local development.

Reply:
Yeah, I think this can be cleaned up a bit. If we just remove the commented out test this solution could apply for the full test folder. But we should probably also separate the e2e tests from the unit tests as we talked about.

But that is another PR.